apps/src/util/testJavabuilderWebsocketConnection.js

```diff
@@ -0,0 +1,58 @@
+// helper to allow behind the scenes test of javabuilder connection
+// on load of high traffic pages. particularly interested
+// in collecting data on whether schools often block websockets.
+export default function testJavabuilderWebsocketConnection() {
+  let token;
+
+  // We log via our own servers to avoid
+  // schools potentially blocking API calls to third parties,
+  // which might affect our results if we were to log to firehose directly.
+  const logEvent = (event, detail) => {
```

Comment:
this should still be a separate function -- it's unclear that it has a hidden dependency on `token` being set before this function is called. we can add `token` as an optional parameter